accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -169,6 +579,11 @@ Windows-specific:
     - `set_MinSendBytesPerSecond(Int64)`
     - `set_RequestQueue(TimeSpan)`
 
+### System.Net.NetworkInformation
+
+* **Ping**
+    - `Send(IPAddress, Int32, Byte[], PingOptions)`
```

Comment:
Is this actually Windows-specific?

Reply:
@dotnet/ncl

Reply:
This is cross-plat, should not be on this list.

Reply:
Removed